internal/resource/service.go

```diff
@@ -184,6 +184,22 @@ func (builder *ServiceBuilder) updatePorts(servicePorts []corev1.ServicePort) []
 			}
 		}
 	}
+	if builder.Instance.TLSEnabled() && builder.Instance.MutualTLSEnabled() {
```

Comment:
The function `MutualTLSEnabled()` already checks if `TLSEnabled()`. Don't we get the same result if we just check for `MutualTLSEnabled()` ?

Reply:
Ah so it does! Will just use the single check here instead. Thanks for pointing that out! Change made here https://github.com/rabbitmq/cluster-operator/pull/451/commits/b737ca4a59b2e05ac2ba106efb485dbe6f603fcd.